CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+# Contributing guide
+
+We're always looking for new contributors and are happy to help you get started! Some ideas:
+
+- Add a test for a function.
```

Comment:
Maybe rephrase it as "find an interesting test case we haven't tested yet" (I trust you to find better wording) to prevent low-effort copy-paste tests from people looking to add open-source contributions to their profiles. We have never had any issues in the past, but I'm afraid of giving people ideas.